frontend/javascripts/oxalis/controller/scene_controller.ts

```diff
@@ -392,13 +392,11 @@ class SceneController {
     const taskBoundingBox = getSomeTracing(state.tracing).boundingBox;
     this.buildTaskingBoundingBox(taskBoundingBox);
 
-    if (state.tracing.volumes.length > 0) {
```

Comment:
the if-check was an optimization which isn't really worth it in my opinion. the instance var types of the classes where not-nullable (which wasn't correct). making them nullable would require a lot of other null-checks in various places. this is why I went with making them actually not-nullable.